### PR TITLE
sig-auth highlights

### PR DESCRIPTION
**Pull request by liggitt**
hoists notable items out of the generated list into the auth section

### CODE REVIEW

**Comment by liggitt** [comment]
cc @kubernetes/sig-auth-pr-reviews

File: release-1.6/release-notes-draft.md [hunk 1]
```diff
@@ -44,6 +44,28 @@ Features for this release were tracked via the use of the [kubernetes/features](
...
+  - default ClusterRole and ClusterRoleBinding objects are automatically updated at server start to add missing permissions and subjects (extra permissions and subjects are left in place). To prevent autoupdating a particular role or rolebinding, annotate it with `rbac.authorization.kubernetes.io/autoupdate=false`. ([#41155](https://github.com/kubernetes/kubernetes/pull/41155), [@liggitt](https://github.com/liggitt))
+  - `v1beta1` RoleBinding/ClusterRoleBinding subjects changed `apiVersion` to `apiGroup` to fully-qualify a subject. ServiceAccount subjects default to an apiGroup of `""`, User and Group subjects default to an apiGroup of `"rbac.authorization.k8s.io"`. ([#41184](https://github.com/kubernetes/kubernetes/pull/41184), [@liggitt](https://github.com/liggitt))
+  - a user can now be authorized to bind a particular role by by having permission to perform the `bind` verb on the referenced role ([#39383](https://github.com/kubernetes/kubernetes/pull/39383), [@liggitt](https://github.com/liggitt))
```
Comment (seh) [inline release-1.6/release-notes-draft.md]: - [ ] Remove extra "by."

Comment (seh) [inline release-1.6/release-notes-draft.md]: Wow, that's confusing to describe. I read #39383, and still pined for an example.

Comment (liggitt) [inline release-1.6/release-notes-draft.md]: added a snippet from the docs with an example to the description of the PR

**Comment by ericchiang** [comment]
lgtm

**Pull request merged**